src/test/java/HeroMapperTest.java

```diff
@@ -1,2 +1,184 @@
+import com.diablo3CharViewer.data_models.FollowerDataModel;
+import com.diablo3CharViewer.data_models.HeroDataModel;
+import com.diablo3CharViewer.data_models.ItemDataModel;
+import com.diablo3CharViewer.data_models.SkillDataModel;
+import com.diablo3CharViewer.json_mappers.HeroMapper;
+import com.diablo3CharViewer.token.FetchToken;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.*;
+
+import static org.mockito.Mockito.lenient;
+
+@ExtendWith(MockitoExtension.class)
 public class HeroMapperTest {
+
+    @Mock
+    private FetchToken testFetchTokenMock;
+    @Mock
+    private HeroMapper testHeroMapperMock;
+    @Mock
+    private Map<String, Integer> heroStatsMock;
+    private final FetchToken tokenAsNull = null;
```

Comment:
Some of those fields are repeated between tests. Can we make them shareable somehow?

Reply:
Make them protected in separate partent class